apps/nextjs-website/react-components/components/common/Common.tsx

```diff
@@ -176,16 +185,27 @@ const ElementBody = (
   variant: TypographyProps['variant'] = 'body2',
   marginBottom: number | string = 0
 ) =>
-  React.isValidElement(body)
-    ? React.cloneElement(
+  React.isValidElement(body) ? (
+    <Box
+      sx={{
+        '& a': {
```

Comment:
```suggestion
        'a': {
```
Same reasoning as above.